plugins/modules/zos_apf.py

```diff
@@ -465,6 +466,7 @@ def main():
 
     arg_defs = dict(
         library=dict(arg_type='str', required=False, aliases=['lib', 'name']),
+        _orig_library=dict(arg_type='str', required=False, aliases=['orig_lib', 'orig_name']),
```

Comment:
we don't need to add an entry into the argument definition, if _orig_library will only contain the original name why not just use a variable?

Reply:
you have a point... I was distracted on Thursday.  I'll rework it today.